src/map/lua/lua_baseentity.cpp

```diff
@@ -17709,6 +17710,98 @@ uint8 CLuaBaseEntity::getMannequinPose(uint16 itemID)
     return 0;
 }
 
+// Fishing Contest Utilities
+
+void CLuaBaseEntity::submitContestFish(uint32 score)
+{
+    if (auto* PChar = dynamic_cast<CCharEntity*>(m_PBaseEntity))
```

Comment:
+1 for correct use of dynamic_cast, probably also want a warning for the cast where this fails (in all bindings too)